nibabel/viewers.py

```diff
@@ -61,6 +61,9 @@ def __init__(self, data, affine=None, axes=None, title=None):
         title : str or None, optional
             The title to display. Can be None (default) to display no
             title.
+        vlim : array-like or None, optional
```

Comment:
What do you think of also allowing strings like `10%` `90%` as input, meaning percentile?  Then the default could be `vlim=('1%', '99%')`.